app/controllers/api/plans_controller.rb

```diff
@@ -46,44 +40,8 @@ def hide
 
   private
 
+  # TODO: where is is used?
```

Comment:
Can we delete this if it's not used?

Reply:
If it's not used, which I believe is the case, we can remove it.

Reply:
OK it's used by its superclass `Api::PlansBaseController`